src/common/import_session.c

```diff
@@ -217,6 +217,14 @@ void dt_import_session_set_filename(struct dt_import_session_t *self, const char
 }
 
 
+/*
```

Comment:
Either remove or put a comments explaining why is this commented out code.

Reply:
BTW, I'm not sure to understand why this is needed on session?

Reply:
@TurboGit, I agree. I mistakenly left this code commented out, but certainly it should be removed.

Reply:
@TurboGit, resolved.